airsonic-main/src/main/webapp/WEB-INF/jsp/albumMain.jsp

```diff
@@ -258,10 +258,15 @@
             <sub:param name="q" value="\"${model.artist}\" \"${model.album}\""/>
             <sub:param name="type" value="album"/>
         </sub:url>
+        <sub:url value="https://www.discogs.com/search/" var="discogsUrl" encoding="UTF-8">
+            <sub:param name="q" value="\"${model.artist}\" \"${model.album}\""/>
```

Comment:
Please escape those values: if there is a `"` in an artist or album, things will go wrong™.

Reply:
Not sure what you mean; how would I better escape them? I tried to just model it off of how the last.fm link was formatted. I just tested the David Bowie album "Heroes" (has quotes in the title) and it seemed to work fine. Also seemed to work when I changed the album metadata so that there is only one quotation mark in the title.

Reply:
Escape via `${fn:escapeXml(model.artist)}` and `${fn:escapeXml(model.album)}`, otherwise, a malicious album/artist name could result in an XSS